src/server/pfs/server/driver_v2.go

```diff
@@ -352,7 +353,11 @@ func (fr *FileReader) drain() error {
 	return nil
 }
 
-func (d *driverV2) compact(master *work.Master, outputPath string, inputPrefixes []string) error {
+type compactResult struct {
```

Comment:
I think `compactStats` might be a better name for this. Also, lets stick with `int64` for the size type across the board.